app/components/channel_list/channel_list.js

```diff
@@ -466,7 +502,9 @@ class ChannelList extends React.Component {
                     </Text>
                 </View>
                 <ListView
-                    ref='scrollContainer'
+                    ref={(ref) => {
```

Comment:
@hmhealey mentioned in my other PR that he may turn on the eslint react/jsx-no-bind rule so might be good to move this to a method.